tests/ha/test_cluster_shutdown_start.py

```diff
@@ -982,10 +971,13 @@ def test_cluster_restart_without_signal(self):
 
         LOGGER.info(
             "Step 2: Start IOs (create s3 acc, buckets and upload objects).")
-        io_resp = self.ha_obj.perform_ios_ops(prefix_data='TEST-29479', nusers=1)
-        assert_utils.assert_true(io_resp[0], io_resp[1])
-        di_check_data = (io_resp[1], io_resp[2])
-        self.s3_clean.update(io_resp[2])
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-29479'
+        self.s3_clean = users
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipcleanup=True)
+        assert_utils.assert_true(resp[0], resp[1])
```

Comment:
May be we can change statement at 981 to "IOs completed successfully". "started" gives impression that IOs are still running.